src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
inverse is an action (verb), better fit for object would be a noun, e.g. inversedRobot